include/ignition/rendering/detail/Export.hh

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
This file is not needed

Reply:
Now no longer created in https://github.com/gazebosim/gz-rendering/pull/618/commits/a4191f722debd816f291558460a23751699cda8e